frontend/app/components/crud-list-item/template.hbs

```diff
@@ -1,36 +1,53 @@
-<tr class="border">
-  <td>
+<tr>
+  <td class="uk-width-1-3">
     {{input-or-show value=entry.key edit=edit placeholder="description"}}
   </td>
-  <td>
+  <td colspan="{{if edit 2 1}}" class="{{if edit 'uk-width-2-3' 'uk-width-1-3'}}">
     {{input-or-show value=entry.value edit=edit placeholder="value" clipboard=true mask=maskPassword}}
   </td>
-  <td class="text-right">
-    {{#if (not edit)}}
-    <button class="btn btn--show" {{action 'show' index}}><i class="fa fa-eye" aria-hidden="true"></i>&nbsp;{{t 'global.show'}}</button>
-    <button class="btn btn--edit" {{action (mut edit) true}}><i class="fa fa-pencil"></i>&nbsp;{{t 'global.edit'}}</button>
-    {{/if}}
-  </td>
+  {{#unless edit}}
+    <td class="uk-width-1-3 uk-text-right">
+        <button {{action 'show' index on='click'}} href="#" class="uk-icon-button" uk-icon="search"></button>
+        <button {{action (mut edit) true on='click'}} href="#" class="uk-icon-button" uk-icon="pencil"></button>
```

Comment:
What happens when you leave out `on='click'`?

Reply:
And why not use the `uk-button` component here?

Reply:
`on='click'` is the default behaviour. I just like being explicit.

Reply:
`uk-button` is an UIkit Button, what we want here is an icon button. Not the same.

Reply:
Alright about the icon.
I don't agree about `on='click'`, to me it's like specifying default parameters. Makes the code brittle and confuses people.

Reply:
I remember that I did this because it was a `<a>` tag before. For a `<button>` tag it's obvious that click is the default action, for `<a>` not so obvious. So I agree that we should remove it here.